packages/layout/tests/steps/resolveBookmarks.test.ts

```diff
@@ -18,7 +18,72 @@ describe('layout resolveBookmarks', () => {
 
     const result = resolveBookmarks(root);
 
-    expect(result).toEqual(root);
+    expect(result).toEqual({
```

Comment:
Unrelated to the fix, but updated this test to test against a new variable instance. The previous test would always pass because the original `root` variable would get mutated.